package/installation.SLES4SAP.xsl

```diff
@@ -108,6 +108,10 @@ textdomain="control"
              	<path>boot/grub2/s390x-emu</path>
              	<archs>s390</archs>
                  </subvolume>
+                 <subvolume>
+             	<path>boot/grub2/arm64-efi</path>
```

Comment:
Indentation is wrong or it is the github visualization?

Reply:
Indentation matches that used throughout that file